core/embed/boardloader/main.c

```diff
@@ -215,6 +218,10 @@ int main(void) {
 
   clear_otg_hs_memory();
 
+#ifdef USE_SDRAM
+  sdram_init();
```

Comment:
Not sure I understand how is the SDRAM used, is it in addition to the on-chip RAM or instead of it? IOW, do we have more memory than on model T?

Reply:
In addition to on chip RAM, this board has external RAM - 8MB more. Its used here for holding frame buffer, which wouldn't fit on chip RAM. (frame buffer is required for display with RGB interface)